docs/book/getting-started/zenml-pro/hybrid-deployment.md

```diff
@@ -0,0 +1,413 @@
+- **VPN/Firewall compatible**: Workspaces operate behind your security perimeter
+- **Audit trails**: Complete logging within your infrastructure
+- **SOC 2 & ISO 27001 certified**: Enterprise-grade security controls
+
+### 🎯 Centralized Governance
+
+- **Unified user management**: Single control plane for all workspaces
+- **Consistent RBAC**: Centrally managed permissions across teams
+- **SSO integration**: Connect with your identity provider once
+- **Global visibility**: Platform teams see across all workspaces
+- **Standardized policies**: Enforce organizational standards
+
+### ⚖️ Balanced Control
+
+- **Infrastructure control**: Full control over workspace configuration and resources
+- **Reduced operational overhead**: ZenML manages the control plane
+- **Customization freedom**: Configure workspaces to specific team needs
+- **Network isolation**: Workspaces can be fully isolated per team/department
+- **Cost optimization**: Pay only for what you use in your infrastructure
+
+### 🚀 Production Ready
+
+- **High availability**: Built-in redundancy for critical components
+- **Automatic updates**: Control plane maintained by ZenML
+- **Professional support**: Direct access to ZenML experts
+- **Monitoring included**: Health checks and alerting configured
+
+## Ideal Use Cases
+
+Hybrid SaaS is perfect for:
+
+- **Regulated industries** (finance, healthcare, government) with strict data residency requirements
+- **Organizations with centralized MLOps teams** managing multiple business units
+- **Companies with existing VPN/firewall policies** that restrict inbound connections
+- **Enterprises requiring audit trails** of all data access within their infrastructure
+- **Teams needing customization** while maintaining centralized user management
+- **Organizations with compliance requirements** mandating on-premises metadata storage
+
+## Architecture Details
+
+### Network Security
+
+#### Outbound-Only Connections
+Workspaces initiate outbound-only connections to the control plane:
+- No inbound connections required to your infrastructure
+- Compatible with strict firewall policies
+
+#### Multi-Workspace Isolation
+Each workspace can be:
+- Deployed in separate VPCs/networks
+- Isolated per team or department or customer
+- Configured with different security policies
+- Managed independently by different teams
+
+### Authentication & Authorization Flow
+
+```mermaid
+graph LR
+    A[User] -->|1. Login| B[Control Plane<br/>ZenML Infrastructure]
+    B -->|2. Auth Token| A
+    A -->|3. Access Workspace| C[Workspace<br/>Your Infrastructure]
+    C -->|4. Validate Token| B
+    B -->|5. Authorization| C
+    C -->|6. Execute| D[Your Resources]
+```
+
+1. User authenticates with ZenML control plane (SSO)
+2. Control plane issues authentication token
+3. User accesses workspace with token
+4. Workspace validates token with control plane
+5. Control plane confirms authorization (RBAC)
+6. Workspace executes operations on your infrastructure
```

Comment:
It's a lot more complicated than this and the authentication flow varies depending on the type of authentication (web client, python client via web login flow, service account, PAT). If the point of this section is to provide a gross oversimplification of the authentication process, you should perhaps get rid of the diagram (not functionally accurate) and simplify the text description to just say that the control plane is in charge of all authentication and authorization, even workspace related.